Problem with building jerryscript on mbed os 5

I try the tutorial below to build jerryscript to an mbed os 5 target.
https://developer.mbed.org/blog/entry/Using-mbed-libraries-with-JerryScript/

In the Setting up section of the tutorial, you can see:

<------------------------
Setting up

The following instructions guide you through the process of setting up an environment where you can run JavaScript on mbed OS:

Open a terminal and navigate to the folder where you cloned the JerryScript repository.
Run cd targets / mbedos5.
Run make getlibs - this will pull in mbed OS and all related repositories.
Run pip install -r ./tools/requirements.txt - this will install Python modules required by the build scripts.
------------------------->
The problem is that when I run the 'make getlibs' command, the following error message appears.

<------------------------
[mbed] ERROR: Could not find mbed program in current path "/ home / parallels / jerryscript / targets / mbedos5".
[mbed] ERROR: Change the current directory to a valid mbed program or use the 'global' option to set global configuration.

mbed config root.
[mbed] ERROR: Could not find mbed program in current path "/ home / parallels / jerryscript / targets / mbedos5".
[mbed] ERROR: Change the current directory to a valid mbed program or use the 'global' option to set global configuration.

make: *** [.mbed] Error 255
------------------------->

So I opened the Makefile and looked at the getlibs configuration section.

<------------------------
getlibs: .mbed

.mbed:
mbed config root.
mbed toolchain GCC_ARM
mbed target $ (BOARD)
mbed deploy
------------------------->

I would like to know what to do if the above error occurs.

Thanks in advance

--- a/targets/mbedos5/.gitignore
+++ b/targets/mbedos5/.gitignore
@@ -2,6 +2,7 @@ mbed-os
 mbed-events
 .build
 .mbed
+.temp/
 mbed_settings.py
 js/pins.js
 source/pins.cpp

--- a/targets/mbedos5/Makefile
+++ b/targets/mbedos5/Makefile
@@ -80,6 +80,7 @@ endif
 getlibs: .mbed
 
 .mbed:
+	echo 'ROOT=.' > .mbed
 	mbed config root .
 	mbed toolchain GCC_ARM
 	mbed target $(BOARD)

